.github/workflows/ci.yaml

```diff
@@ -5,18 +5,37 @@ on:
     branches: '**'
 
 jobs:
+  get-go-version:
+    name: "Determine Go toolchain version"
+    runs-on: ubuntu-latest
+    outputs:
+      go-version: ${{ steps.get-go-version.outputs.go-version }}
+    steps:
+      - uses: actions/checkout@c85c95e3d7251135ab7dc9ce3241c5835cc595a9 # v3.5.3
+      - name: Determine Go version
+        id: get-go-version
+        # We use .go-version as our source of truth for current Go
+        # version, because "goenv" can react to it automatically.
+        run: |
+          echo "Building with Go $(cat .go-version)"
+          echo "::set-output name=go-version::$(cat .go-version)"
```

Comment:
**suggestion:** This `set-output` is no longer the correct way to do this with GHA. It will eventually stop working.

```
- name: Determine Go version
        id: get-go-version
        # We use .go-version as our source of truth for current Go
        # version, because "goenv" can react to it automatically.
        run: |
          echo "Building with Go $(cat .go-version)"
          echo "go-version=$(cat .go-version)" >> "${GITHUB_OUTPUT}"
 ```

Reply:
Ah, I lifted this from `consul`/`consul-k8s` - thank you for catching. Do we need to update there as well now, or is that covered by some ticket somewhere?

Reply:
It is in a ticket somewhere but isn't as much of a priority as 99% of the workflows are in consul-k8s-workflows.

Reply:
Got it (and yeah, `consul` would be the more concerning of the two, if we didn't have that covered)